Updates/0703_escort.sql

```diff
@@ -0,0 +1,169 @@
+(@PATH*100+3,0,4,10,8337,300000,4,0,0,0,8,0,0,0,0,-6805.320, -1806.258, 261.357, 0.42,0,0,'Dorius Stonetender - Spawn Dark Iron Steelshifter'),
+(@PATH*100+3,0,5,40,0,0,0,147517,10120,17,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Despawn DarkIron GO'),
+(@PATH*100+3,0,6,40,0,0,0,147517,10127,17,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Despawn DarkIron GO'),
+(@PATH*100+3,2000,0,28,8,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - State Kneel'),
+(@PATH*100+3,4000,1,28,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Remove State Kneel'),
+(@PATH*100+4,3000,0,25,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Set Run'),
+(@PATH*100+4,4000,1,0,0,0,0,0,0,0,4357,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Say Text'),
+(@PATH*100+5,0,0,20,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Change Movement'),
+(@PATH*100+5,0,1,0,0,0,0,0,0,0,4363,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Say Text'),
+(@PATH*100+5,1000,0,10,8338,10000,0,0,0,0,8,0,0,8338,0,-6369.839,-1974.8733,256.84335,3.5683,0,0,'Dorius Stonetender - Spawn Dark Iron Marksman');
+
+-- Dorius Stonetender rp script when escort is finished
+SET @RELAYID := 8284;
+DELETE FROM `dbscripts_on_relay` WHERE id = @RELAYID;
+INSERT INTO dbscripts_on_relay(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+(@RELAYID,0,0,0,0,0,0,0,0,0,4359,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Say Text'),
+(@RELAYID,0,1,15,12218,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Cast Spell'),
+(@RELAYID,1000,0,0,0,0,0,0,0,0,5944,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Say Text'),
+(@RELAYID,1000,1,9,10106,300,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Respawn Object'),
+(@RELAYID,1000,2,21,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Set Active'),
+(@RELAYID,1000,3,28,7,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Emote Death'),
+(@RELAYID,241000,0,21,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Remove Active'),
+(@RELAYID,241000,1,18,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Despawn Self');
+
+-- Dark Iron Marksman relayscript OnSpawn
+SET @RELAYID := 8338;
+DELETE FROM creature_spawn_data_template WHERE Entry = @RELAYID;
+INSERT INTO `creature_spawn_data_template` (`Entry`, `RelayId`, `Name`) VALUES (@RELAYID, @RELAYID, 'Dark Iron Marksman - RelayScript on Spawn');
+DELETE FROM `dbscripts_on_relay` WHERE id = @RELAYID;
+INSERT INTO dbscripts_on_relay(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+(@RELAYID,0,0,0,0,0,0,0,0,0,4358,0,0,0,0,0,0,0,0,0,0,'Dark Iron Marksman - Say Text'),
+(@RELAYID,1000,0,15,12198,0,0,8284,30,0,0,0,0,0,0,0,0,0,0,0,0,'Dark Iron Marksman - Cast Spell'),
```

Comment:
I think thats the only thing thats not working correctly atm.
The spell gets casted from Dark Iron Marksman (animation) and needs to hit Dorius Stonetender.
currently the animation on dark iron marksman is missing but dorius gets hited.
if we use SCRIPT_FLAG_BUDDY_AS_TARGET, dark iron marksman has the cast animation but the spell dont hit dorius anymore so the rp (core OnSpellHit) dont get triggert